sys/storage/nvs/nvs.c

```diff
@@ -28,18 +28,15 @@
  */
 
 
-#define ENABLE_DEBUG (1)
 #include "debug.h"
```

Comment:
This needs to be at the bottom too. in the same order:

```
#define ENABLE_DEBUG (0)
#include "debug.h"
```

Otherwise you'll get double definitions errors.